### PR TITLE
feat : added a grid component

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

**Review by MiheerTamkhane — APPROVED** [review]
All looks good just remove comments, Good to merge.

File: Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+</head>
+<body>
+
+    <nav>
+        <div class="nav-section left-nav-links">
+          <div>
+            <a class="nav-a-link" href="../index.html"> Geeta UI</a>
+           </div>
+        </div>
+
```
Comment (Varun-28) [inline Grid/grid.html:23]: - remove redundant spaces in the Html file
- use prettier extension for proper code formatting for your HTML and CSS files
- Rest things are clean and good to go

File: Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                    } </code>
+                
+                  </pre>
+                  
+                </div>
+    
+
+
+
+
```
Comment (Jayesh3636) [inline Grid/grid.html:200]: Remove the unnecessary spaces!

File: Grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+             <br/>
+            <hr/>
+
+
+
+
+
+
```
Comment (Jayesh3636) [inline Grid/grid.html:208]: same here

File: Grid/grid.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+                 </li>
+             </ul>
+       
+     </div>
+    </nav>
+    
+    <br> <br> 
+
+
+
```
Comment (Jayesh3636) [inline Grid/grid.html:47]: unnecessary spaces!

File: Grid/grid.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+            <div class="content-title"> CSS</div>
+
+            <div class="code-content-div"> 
+           
+                <pre>
+                  <code class="code">           
+                    .grid--col-2{
+                        display:grid;
+                        grid-template-columns: 1fr 1fr;
+                        grid-column-gap:20px;
```
Comment (Jayesh3636) [inline Grid/grid.html:119]: try to use rems if the value is great than10px.

**Review by Jayesh3636 — APPROVED** [review]
Just mentioned a few changes in comments rest good to go

**Pull request merged**